src/app/pages/send/send-crypto-asset-form/family/bitcoin/hooks/use-send-max.tsx

```diff
@@ -23,19 +23,20 @@ export function useSendMax({
 }: UseSendMaxArgs) {
   const [, _, amountFieldHelpers] = useField('amount');
   const [, __, feeFieldHelpers] = useField('fee');
+  const toast = useToast();
 
   const analytics = useAnalytics();
 
   return useCallback(() => {
     void analytics.track('select_maximum_amount_for_send');
     if (balance.amount.isLessThanOrEqualTo(0)) {
-      toast.error(`Zero balance`);
+      toast.error('Zero balance');
```

Comment:
Not for this PR but it seems odd that sometimes we use form validation and other times we show a toast. Maybe we should just use form validation?

Also curious why this is nested under a `family` folder

Reply:
Good question, not sure why it is located here. I can check and move.

Reply:
Also, hoping your first question is addressed in redesign of send flow.